src/components/payment/PaymentToggleModel.tsx

```diff
@@ -0,0 +1,54 @@
+import { useEffect } from 'react';
+import momoIcon from '../../assets/momo.png';
+import PaymentButton from '../buttons/PaymentButton';
+import { useAppSelector, useAppDispatch } from '../../redux/hooks/hooks';
+import { stripePayment } from '../../redux/features/StripePaymentSlice';
+import stripeIcon from '../../assets/stripe icon.png';
+import cancel from '../../assets/cancel.png';
+import { payModel } from '../../redux/features/toggleSlice';
+
+function PaymentToggleModel() {
+	const { data, isLoading } = useAppSelector((state) => state.stripe);
```

Comment:
Morning @Angemichel12 , let me hope you are doing great. I have a simple request about stripe payment

<img width="1440" alt="Screenshot 2024-07-23 at 09 05 54" src="https://github.com/user-attachments/assets/133ad9cc-098a-4f97-9e02-8e92d25109fd">

Can you catch this error, so that a user can know what happened instead of just showing him/her that there is a request with 400 status, he may not even know what that is. I think you can improve that then we're good